activejob/lib/active_job/queue_adapters/qu_adapter.rb

```diff
@@ -23,7 +23,7 @@ def enqueue(job, *args) #:nodoc:
       end
 
       def enqueue_at(job, timestamp, *args) #:nodoc:
-        raise NotImplementedError
+        raise NotImplementedError.new("This queueing backend doesn't not support scheduling jobs. To see what features are supported see http://api.rubyonrails.org/classes/ActiveJob/QueueAdapters.html")
```

Comment:
can we skip the `.new` and use this `raise` syntax:

```ruby
raise NotImplementedError, "message"
```

Did a quick grep and that's what most of Rails uses.